astropy/modeling/blackbody.py

```diff
@@ -48,6 +48,26 @@
 >>> flux_nu  # doctest: +FLOAT_CMP
 <Quantity [  4.25135927e-123,  2.36894060e-005] erg / (cm2 Hz s sr)>
 
+Alternatively, the same results for ``flux_nu`` can be computed using ``BlackBody1D``
+from :class:`BlackBody1D` with blackbody representation as a model. The difference between
+this and the former approach is in one addition step as outlined as follows:
+
+>>> import numpy as np
+>>> from astropy import constants as const
+>>> from astropy import units as u
+>>> from astropy.modeling import models
+>>> temperature = 5000 * u.K
+>>> extra = const.sigma_sb * temperature ** 4 / np.pi
+>>> extra.cgs  # doctest: +FLOAT_CMP
+<Quantity 1.12808367e+10 g / s3>
+>>> wavelengths = [100, 10000] * u.AA
+>>> bb_astro = models.BlackBody1D(temperature, bolometric_flux=extra)
+>>> bb_astro(wavelengths).cgs  # doctest: +FLOAT_CMP
```

Comment:
Printing it back out in the exact same unit as `flux_nu` would be better in illustrating that this is indeed equivalent. The doctest output below would also need to be updated.
```suggestion
>>> bb_astro(wavelengths).to(u.erg / (u.cm * u.cm * u.Hz * u.s * u.sr))  # doctest: +FLOAT_CMP
```